convert codebase from Py2 to Py3; use black for formatting

prusnak opened the pull request:
I used two tools to convert the codebase from Py2 to Py3:

the first one was "2to3", the other was "black" formatter

The result is untested!!! Please test before the merge

djmelik commented:
Thanks, checking now.

djmelik commented:
This is interesting, I'm no longer able to import rpc_pb2_grpc in python3. However, it works if I move rpc_pb2.py to the base directory. Doing some research. 

```
$ python
Python 3.6.5 (default, Apr 25 2018, 14:23:58)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from libs import rpc_pb2 as ln
>>> from libs import rpc_pb2_grpc as lnrpc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/melik/dev/lndash/libs/rpc_pb2_grpc.py", line 4, in <module>
    import rpc_pb2 as rpc__pb2
ModuleNotFoundError: No module named 'rpc_pb2'
```

djmelik commented:
It appears that relative imports were removed in python3.3+? I believe that is why it's failing.

Changing line 4 on libs/rpc_pb2_grpc.py seems to fix it: 
```
>import rpc_pb2 as rpc__pb2
<from . import rpc_pb2 as rpc__pb2
```

prusnak commented:
Updated commit to include this change. I also removed futures from requirements, because this is not needed for Python 3 (it is a feature backport module).
